packages/gatsby-plugin-gatsby-cloud/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "gatsby-plugin-gatsby-cloud",
+  "description": "A Gatsby plugin which optimizes working with Gatsby Cloud",
+  "version": "1.1.0-next.0",
+  "author": "Kyle Mathews <mathews.kyle@gmail.com>",
+  "bugs": {
+    "url": "https://github.com/gatsbyjs/gatsby/issues"
+  },
+  "dependencies": {
+    "@babel/runtime": "^7.12.5",
+    "fs-extra": "^8.1.0",
+    "kebab-hash": "^0.1.2",
+    "lodash": "^4.17.20",
+    "webpack-assets-manifest": "^3.1.1"
+  },
+  "devDependencies": {
+    "@babel/cli": "^7.12.1",
+    "@babel/core": "^7.12.3",
+    "babel-preset-gatsby-package": "^1.0.0-next.0",
+    "cross-env": "^7.0.3",
+    "jest": "^26.0.1"
```

Comment:
this probably should be removed, our other package don't add `jest` directly and rely on `jest` provided by root of monorepo

Reply:
Removed